src/app/frontend/login/component.ts

```diff
@@ -65,6 +65,8 @@ export class LoginComponent implements OnInit {
       .get<EnabledAuthenticationModes>('api/v1/login/modes')
       .subscribe((enabledModes: EnabledAuthenticationModes) => {
         this.enabledAuthenticationModes_ = enabledModes.modes;
+        this.enabledAuthenticationModes_.push(LoginModes.Kubeconfig);
```

Comment:
I think we can simplify the logic a bit here. Even if enabled modes will be empty, then we push kubeconfig at the end and select first option from the list as active one.

Reply:
Excellent! Thanks!!